src/libecalc/fixtures/cases/consumer_system_v2/data/consumer_system_v2.yaml

```diff
@@ -125,16 +124,15 @@ INSTALLATIONS:
             ENERGY_USAGE_MODEL: compressor_sampled_1d
           - NAME: compressor3
             ENERGY_USAGE_MODEL: compressor_sampled_1d
+        COMPONENT_CONDITIONS:
+          CROSSOVER: [ 0, 1, 1 ]
         OPERATIONAL_SETTINGS:
-          - RATES: [ 1000000, 5000000, 5000000 ]
+          - RATES: [ 1000000, 6000000, 6000000 ]
```

Comment:
We should look into restructuring this, do the users prefer thinking in indices or should we just use a map where the key is the compressor name? 

```yaml
          CONSUMER_STREAM_CONDITIONS:
            - consumer1:
                STREAMS:
                  - RATE: 1000000
                    INLET_PRESSURE: 50
                    OUTLET_PRESSURE: 250
              consumer2:
                STREAMS:
                  - RATE: 6000000
                    INLET_PRESSURE: 50
                    OUTLET_PRESSURE: 250
              consumer3:
                STREAMS:
                  - RATE: 6000000
                    INLET_PRESSURE: 50
                    OUTLET_PRESSURE: 250
```

if we should support multiple streams, if not

```yaml
          CONSUMER_STREAM_CONDITIONS:
            - consumer1:
                STREAM:
                  RATE: 1000000
                  INLET_PRESSURE: 50
                  OUTLET_PRESSURE: 250
              consumer2:
                STREAM:
                  RATE: 6000000
                  INLET_PRESSURE: 50
                  OUTLET_PRESSURE: 250
              consumer3:
                STREAM:
                  RATE: 6000000
                  INLET_PRESSURE: 50
                  OUTLET_PRESSURE: 250
```

Reply:
I think something like this is better, since it does not give the cognitiv overload of making sure that the index that you add sth to is correct, pretty sure it is easy to do a mistake...

however, quite much more verbose when writing yaml...so there is a trade off... also topic for discussion..